pkg/cloudprovider/providers/gce/gce_util.go

```diff
@@ -153,3 +153,20 @@ func isNotFoundOrInUse(err error) bool {
 func makeGoogleAPINotFoundError(message string) error {
 	return &googleapi.Error{Code: http.StatusNotFound, Message: message}
 }
+
+func isForbidden(err error) bool {
+	return isHTTPErrorCode(err, http.StatusForbidden)
+}
+
+func handleNetworkTierGetError(err error) (string, error) {
```

Comment:
is this logic not possible to just put in the GetAlphaRegionAddress, ... functions?

this is a super generic function name...

Reply:
It's shared by two files -- gce_addresses.go and gce_forwardingrule.go.
I renamed it to `handleAlphaNetworkTierGetError()` to make it slightly more explicit, and also added a TODO with a bug number to remove this.